hello-libs/settings.gradle

```diff
@@ -1,9 +1,2 @@
 include ':app'
 
-// If you want to generate libs, just uncomment out the following
-// and dependency in app/build.gradle
-
-// include ':gen-libs'
-
-
-
```

Comment:
thanks for PR, may you resolve the conflict and comments?  thank you for your efforts to make the repo better!